src/arns.lua

```diff
@@ -415,10 +421,13 @@ end
 local function getTotalReturnedNamesBetweenTimestampsUnsafe(startTimestamp, endTimestamp)
 	local returnedNames = arns.getReturnedNamesUnsafe()
 	local totalReturnedNames = 0
-	local returnedNameEndTimestamp = startTimestamp + constants.returnedNamePeriod
 
 	for _, returnedName in pairs(returnedNames) do
-		if startTimestamp >= returnedName.startTimestamp and endTimestamp <= returnedNameEndTimestamp then
+		-- returned name is active between the start and end timestamps
+		if
+			returnedName.startTimestamp <= startTimestamp
+			and returnedName.startTimestamp + constants.returnedNamePeriod >= endTimestamp
```

Comment:
no change in logic here - just standardizing them, so it reads as : "X started BEFORE startTimestamp, and ends AFTER endTimestamp)"